Modules/Loadable/VolumeRendering/MRML/vtkMRMLVolumeRenderingDisplayNode.h

```diff
@@ -114,6 +115,15 @@ class VTK_SLICER_VOLUMERENDERING_MODULE_MRML_EXPORT vtkMRMLVolumeRenderingDispla
   vtkBooleanMacro(AutoClippingBlankVoxelValue, bool);
   //@}
 
+  //@{
+  /// Check if the clipping shortcut can be used with the display/clipping node.
+  /// Returns true if the clipping shortcut can be utilized, or returns false otherwise.
```

Comment:
In general, I would avoid using "shortcut" word, as it is not very clear. The important thing is that if the shortcut can be used then the clipping will be _fast_.